iceoryx_utils/include/iceoryx_utils/internal/cxx/string.inl

```diff
@@ -0,0 +1,243 @@
+// Copyright (c) 2019 by Robert Bosch GmbH. All rights reserved.
```

Comment:
year

Reply:
this was actually done in a branch in 2019, it just happens that it got ready this year